### PR TITLE
Adding r6g EC2 instance types

### PR DESCRIPTION
**Pull request by blee-wd**
Hoping to add r6g EC2 instance types

### CODE REVIEW

**Comment by blee-wd** [comment]
I signed it

**Comment by blee-wd** [comment]
I signed it

File: cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go [hunk 1]
```diff
@@ -1930,4 +1930,34 @@ var InstanceTypes = map[string]*InstanceType{
...
+		MemoryMb:     393216,
+		GPU:          0,
+	},
+	"r6g.16xlarge": {
+		InstanceType: "r6g.16xlarge",
+		VCPU:         64,
+		MemoryMb:     524288,
+		GPU:          0,
+	},
+	"r6g.24xlarge": {
```
Comment (gjtempleton) [inline cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go:1945]: Where is this `r6g.24xlarge` information coming from, as I can't seem to find any information on it, and run `make generate doesn't seem to pull this information in for me.

**Comment by Jeffwan** [comment]
/approve cancel

@gjtempleton  I think you are right, I didn't check this clearly. This is not in the list.  https://aws.amazon.com/ec2/pricing/on-demand/

As #3328 covers this PR, I think we can close it. @blee-wd Thanks for your contribution. If you have further questions, feel free to reopen the PR

**Pull request closed without merging**